tests/slow/src/test/scala/tests/feature/BracingSwitchCodeActionLspSuite.scala

```diff
@@ -0,0 +1,108 @@
+package tests.feature
+
+import munit.{Location, TestOptions}
+import tests.codeactions.BaseCodeActionLspSuite
+
+import scala.meta.internal.metals.BuildInfo
+import scala.meta.internal.metals.codeactions._
+import scala.meta.internal.mtags.MtagsEnrichments.XtensionAbsolutePath
+
+class BracingSwitchCodeActionLspSuite
+    extends BaseCodeActionLspSuite("cross-code-actions") {
+
+  override protected val scalaVersion: String = BuildInfo.scala3
+
+  check(
+    "braceful-to-braceless-object",
+    """|object Ma<<>>in {
+       |  def method2(i: Int) = ???
+       |}
+       |
+       |class A
+       |""".stripMargin,
+    s"""|${ExtractRenameMember.title("object", "Main")}
+        |${BracelessBracefulSwitchCodeAction.goBraceless}""".stripMargin,
```

Comment:
Because `BracelessBracefulSwitchCodeAction` is listed as a second one you have to specify `selectedActionIndex` for the `check` method. In your case it is `selectedActionIndex = 1`

Reply:
Thank you so much :)

Reply:
It was caused by not copying `l.Position` through new instance creating and assigning the right character and line numbers. I was instead just assigning the old Position to a new val, which was pass by reference. So when I was editing the character of the end position for example, that of the start position was also getting modified as a side effect.